extensions/ql-vscode/src/databases/db-config.ts

```diff
@@ -1,10 +1,20 @@
 // Contains models for the data we want to store in the database config
 
 export interface DbConfig {
+  databases: DbConfigDatabases;
+  selected?: SelectedDb;
+}
+
+export interface DbConfigDatabases {
   remote: RemoteDbConfig;
   local: LocalDbConfig;
 }
 
+export interface SelectedDb {
+  kind: string;
```

Comment:
Let's add some more specific types here rather than just strings e.g. `kind` can be an enum with two possible values.

Reply:
Ok, I wasn't sure how to exactly flesh this out in the later PRs, that's why I left it as open as possible